src/sounds.cpp

```diff
@@ -991,11 +991,11 @@ void sfx::do_vehicle_engine_sfx()
         current_gear = previous_gear;
     }
 
-    if( current_gear > previous_gear ) {
+    if( !veh->is_autodriving && current_gear > previous_gear ) {
```

Comment:
nit: Wrap this in a big `if` instead of inserting it into each branch. Eg.
```suggestion
    if( !veh->is_autodriving ) {
        if( current_gear > previous_gear ) {
```
...